puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-ci-centos-simple-test.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Do we want to keep this or was it just for debugging?

Reply:
I would like to keep it. This gave an easy (and quick) way to test the interaction without having to run any one of our production level jobs.